### ISSUE TITLE
[Bug]: Reducing the CVSS Score below 3.9 doesn't impact the severity

### ISSUE THREAD

**Issue by fersy02** [apisec-inc/Support#151]
### Severity

Blocker

### What happened?

Per the CVSS Severity ratings, reducing the score to 3.9 or lower should reduce the severity to Low. However, it has been noticed that reducing the CVSS score below 3.9 is still filing the vulnerabilities with Medium severity instead of Low.
![image](https://user-images.githubusercontent.com/47815523/189737692-7908f6b9-d289-4738-81cb-df06eb1f728c.png)

Several clients we work with have raised concerns about this behavior as they couldn't go to production with medium-severity vulnerabilities. In general, vulnerabilities with a  lower impact, such as stored injection, should be allowed to file as low-severity vulnerabilities.

I have reproduced this issue in the NetBanking Application.

Steps to reproduce:
1. Register and configure the NetBanking API
2. Navigate to categories and select a category such as Sensitive Data Exposure L2
3. Modify the Vulnerability scoring system to custom, update the score to 2.8 and click Save
4. Run a scan by selecting the Sensitive Data Exposure L2 category

Expected Result:
Vulnerabilities should be filed as low-severity

Actual Result:
Vulnerabilities are filed as medium-severity

Reference:
https://nvd.nist.gov/vuln-metrics/cvss
![image](https://user-images.githubusercontent.com/47815523/189739019-f986b894-67a9-4c08-9081-b8a5108e5773.png)

**Comment by mmuqtaar** [comment]
@fersy02 As per instruction, we are not setting it to low tag. We have only Critical, High & Medium Tags.

**Comment by atef-aa** [comment]
@fersy02 - removing the blocker tag as this is not blocking a behavior. This should be picked up by the next available engineer.

**Comment by fersy02** [comment]
@atef-aa 
The client was blocked for a Go Live as they have a policy for 0 Critical/High/Medium vulnerabilities. We had to temporarily archive those vulnerabilities as we couldn't file them as Low-severity vulnerabilities and help them move forward.

**Comment by amer-apisec** [comment]
verified in dev, not working pls refer screenshot.

![image](https://user-images.githubusercontent.com/105206963/191191464-2bc75d6f-02f8-4da0-ac9e-b1984a0c8476.png)

![Screenshot 2022-09-20 124008](https://user-images.githubusercontent.com/105206963/191191902-78cb9eb8-da8c-4043-9470-99fb30878ccc.png)

**Comment by amer-apisec** [comment]
verified in dev, Score below 3.9 still showing as medium severity, pls refer screenshot.

![image](https://user-images.githubusercontent.com/105206963/192199271-e94aace4-6478-49b0-8b8e-638321e995d4.png)

**Comment by mmuqtaar** [comment]
@amer-apisec I have verified in dev env, It's working reproduce step 

![image](https://user-images.githubusercontent.com/87175886/192202052-43b8d191-8c2c-48da-9d51-c126a350ea5f.png)

**Comment by amer-apisec** [comment]
**Steps to reproduce**
Test case - 1
1. Go to Security categories from the main dashboard page
2.  change unsecured category cvss to 3.3
3.  click on Rebuild and launch
4.  now register a new project and hit scan for unsecured 
5. still the 3.3 is showing as medium severity

![image](https://user-images.githubusercontent.com/105206963/192206146-5e4083ac-e2e8-4cad-82da-360db2707db0.png)

Test case - 2
1. Register  a new project 
2. we can see the change in score (3.3) of unsecured in project security categories
3. click save and rewrite playbooks button
4. scan unsecured category
5. the score (3.3) is showing as low severity

![image](https://user-images.githubusercontent.com/105206963/192207026-df390447-9b0d-463f-9c0f-d00056c555b2.png)

**Comment by mmuqtaar** [comment]
@amer-apisec Issue is resolved to unblock the customer as per test case-2. However, you can close the issue & open a new ticket for test case-1

**Comment by muqeetkhan20** [comment]
@mmuqtaar the new project creation should not force us to save and re-write the playbooks to impact the severity.
Customer may treat this as a bug. Please resolve this.
![image (16)](https://user-images.githubusercontent.com/68276700/192710461-56183314-9653-4807-a933-da58b82f51fe.png)

**Comment by mmuqtaar** [comment]
@muqeetkhan20 It is currently in our design, we cannot make changes to it.

**Comment by muqeetkhan20** [comment]
Discussed with @Mohammed-Luqman-Shareef, it's a valid test case

**Comment by muqeetkhan20** [comment]
@mmuqtaar we need to make this fully functional.
One of the scenario which is significant is not working. I have already discussed with @Mohammed-Luqman-Shareef; it seems to be a valid scenario. We can close this ticket once the said scenario is fixed.

**Comment by YasmeenFatima** [comment]
Its working fine is dev.
![image](https://user-images.githubusercontent.com/69448810/196903848-0402578b-386b-445b-9d90-b4d10efb2b85.png)

**Comment by mmuqtaar** [comment]
As per this ticket, functionality is delivered. Close this ticket.
